composeApp/build.gradle.kts

```diff
@@ -76,4 +75,7 @@ android {
         debugImplementation(libs.compose.ui.tooling)
     }
 }
+dependencies {
```

Comment:
What's the difference between this dependency block and the androiMain.Dependencies block?

Reply:
Not sure, I moved it to the androidMain and it works fine, not sure why I put it there 😅